test/unit-tests/libraries/Message.t.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.9;
+
+/// library imports
+import {Test, Vm} from "forge-std/Test.sol";
+
+/// local imports
+import "src/libraries/Message.sol";
+
+/// @dev is a helper function to test library contracts
+contract MessageHelper {
+    using MessageLibrary for MessageLibrary.Message;
+
+    function computeMsgId(MessageLibrary.Message memory _message) external pure returns (bytes32) {
+        return _message.computeMsgId();
+    }
+
+    function extractExecutionParams(MessageLibrary.Message memory _message)
+        external
+        pure
+        returns (MessageLibrary.MessageExecutionParams memory)
+    {
+        return _message.extractExecutionParams();
+    }
+
+    function computeExecutionParamsHash(MessageLibrary.MessageExecutionParams memory _params)
+        external
+        pure
+        returns (bytes32)
+    {
+        return MessageLibrary.computeExecutionParamsHash(_params);
+    }
+
+    function computeExecutionParamsHashFromMessage(MessageLibrary.Message memory _message)
+        external
+        pure
+        returns (bytes32)
+    {
+        return _message.computeExecutionParamsHash();
+    }
+}
```

Comment:
Unless I am missing something, this contract feels unnecessary. Suggest refactoring as follows:
- remove this helper contract
- define `using MessageLibrary for MessageLibrary.Message` in `MessageLibraryTest` contract
- directly call functions in MessageLibrary instead of through a separate helper (e.g. `message.computeMsgId()` in place of `MessageHelper.computeMsgId(message)`

Reply:
I see, interesting! 
- I'd suggest adding a comment and a link to [the relevant issue in Foundry that mentions this](https://github.com/foundry-rs/foundry/issues/2567), so we retain context about why we added this contract and can change this in the future if that issue is ever addressed. 
- I'd also suggest changing the name of the contract to something like "MessageLibraryTestClient" instead of helper, to make the intent a bit clearer